packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/lib/getDrupalSearchResults.js

```diff
@@ -0,0 +1,30 @@
+import { defaultFetch } from '@pantheon-systems/drupal-kit';
+
+/**
+ * @description Helper function to query the Drupal Search API.
+ * @param {import('next').GetServerSidePropsContext} context - Nextjs getServerSideProps context.
+ * @param {string} index - The specific Drupal Search index to be used in the query. Default is an example 'articles_index'.
+ * @see https://www.drupal.org/docs/contributed-modules/search-api for more information about the Drupal Search API.
+ * @returns An array of search results matching the users search term.
+ */
+
+export const getDrupalSearchResults = async (
+	context,
+	index = 'articles_index',
```

Comment:
```suggestion
       {
         query,
         locale,
         res
         index = 'articles_index',
       }
```
I think I favor explicit params here. I see we have some functions that take just `context` but I wonder if those should be updated at some point as well.

Reply:
This doesn't seem to have been addressed– The ask here is to use named arguments. It's probably cleanest to use an interface

```ts
interface GetDrupalSearchResultsParams {
	apiUrl: string;
	locale: string;
	query: string;
	response: ServerResponse;
	index: string;
}

//...tsdoc stuff
const getDrupalSearchResults = async ({
	apiUrl,
	locale,
	query,
	response,
	index = 'articles_index',
}: GetDrupalSearchResultsParams) => {